binaries/cpp-load-from-git

```diff
@@ -0,0 +1 @@
+https://github.com/DataDog/dd-trace-cpp@07aa32bf395bf3bd1e4f57f124a21b564bd932fe
```

Comment:
DO NOT MERGE THIS FILE. This is to just test this change in CI. This change will reverted soon.